test/e2e/revision_test.go

```diff
@@ -48,6 +57,30 @@ func TestRevision(t *testing.T) {
 	})
 }
 
+func (test *e2eTest) revisionListWithService(t *testing.T, serviceNames ...string) {
+	for _, svcName := range serviceNames {
+		confGenStr := test.findConfigurationGeneration(t, svcName)
+		confGen, err := strconv.Atoi(confGenStr)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		out, err := test.kn.RunWithOpts([]string{"revision", "list", "-s", svcName}, runOpts{})
+		assert.NilError(t, err)
+
+		outputLines := strings.Split(out, "\n")
+		// Ignore the last line because it is an empty string caused by splitting a line break
+		// at the end of the output string
+		for _, line := range outputLines[1 : len(outputLines)-1] {
+			// Skip checking of a revision name line-by-line because currently we don't have a good way to capture revision names
+			// randomly generated by server side
+			// TODO(toVersus): check revision name once `kn service update --revision-name revName` is merged
```

Comment:
don't get that comment

Reply:
I removed this comment and changed to validate a revision name line-by-line.